packages/atomic/src/components/atomic-result-list-v1/atomic-result-list-cards.pcss

```diff
@@ -1,9 +1,14 @@
 @define-mixin atomic-list-with-cards {
   /* TODO: Remove v1 suffix */
-  atomic-result-v1 {
+  atomic-result-v1,
+  atomic-result-placeholder-v1 {
     border: 1px solid var(--atomic-neutral);
     margin: 1rem;
     padding: 1rem;
     border-radius: 1rem;
   }
+
+  atomic-result-placeholder-v1 {
+    border-color: transparent;
+  }
```

Comment:
Rather than removing borders and dividers, I opted to hide them. I want to keep the paddings/gaps as similar as possible.